packages/python/plotly/plotly/tests/test_core/test_graph_objs/test_performance.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import time
+from unittest import TestCase
+import pytest
+import numpy as np
+import plotly.graph_objs as go
+import plotly.io as pio
+from plotly.tests.b64 import b64, _b64
+
+
+def test_performance_b64():
+    rand_arr_1 = np.random.random(100000)
+    rand_arr_2 = np.random.random(100000)
+    raw_arr_1 = rand_arr_1.tolist()
+    raw_arr_2 = rand_arr_2.tolist()
+    b64_arr_1 = b64(rand_arr_1)
+    b64_arr_2 = b64(rand_arr_2)
```

Comment:
Do we need `b64` conversion here?
Can't we simply pass `rand_arr_1` `numpy` input data?

Reply:
I'm confused - that variable is a numpy array, so it wouldn't be in the b64 format?

Reply:
Exactly. #4470 is about passing numpy arrays directly. So yes we shouldn't need to cast to b64 here.